README.md

```diff
@@ -20,10 +20,12 @@ setup:
 - Install Docker, and make sure that the current user is in the `docker` group.
 - Install jq
 - (optionally) Install tmux -- this only needed if you want to use it.
+- Install Docker Buildx (required for building Docker images)
 
 On Ubuntu you can do this with:
 
     sudo apt install git docker.io jq tmux
+    sudo apt install docker-buildx
```

Comment:
We could, more economically, add `docker-buildx` to L27 instead of using a separate `apt install` invocation.

Reply:
Nice, thanks very much.

In the meantime, @Arnaud-de-Grandmaison-ARM has found an issue (and a fix) for a build issue that breaks the CI, see #290 

When that PR is merged, you'll need to rebase your branch on top of `main`.

Reply:
Thank you for the update, @thomas-fossati! I’ll keep an eye on #290 and will rebase my branch on top of the main branch once the fix is merged.

Reply:
@thomas-fossati  , I'm currently exploring some issues within the community to get more familiar with the organization. I've checked many repositories, and I've noticed that you are one of the most active mentors here. I would greatly appreciate it if you could suggest any specific issues or areas where I could contribute to improve and make a meaningful impact.
Thank you!

Reply:
> I would greatly appreciate it if you could suggest any specific issues or areas where I could contribute to improve and make a meaningful impact.

It depends on your skills and the amount of time you can commit.

I suggest joining https://veraison.zulipchat.com so we can have a one-on-one chat there.